pipeline/00-ingest.R

```diff
@@ -290,6 +292,34 @@ training_data_clean <- training_data_w_hie %>%
     ccao_is_corner_lot = replace_na(ccao_is_corner_lot, FALSE),
     across(where(is.character), \(x) na_if(x, ""))
   ) %>%
+  # Get a count of the number of sales that have occurred in the last n years
+  left_join(
+    left_join(
+      training_data %>%
+        select(meta_pin, meta_sale_document_num, meta_sale_date),
+      training_data %>%
+        filter(!sv_is_outlier) %>%
```

Comment:
I think we probably only want the count of non-outlier sales.

Reply:
This I'm not positive about. I could see us also wanting to include outliers if we're trying to detect flips, i.e. two quick sales where the first would be considered an outlier and the second isn't.